src/types/reaction-added.ts

```diff
@@ -13,4 +13,13 @@ export interface Item {
   ts?: string;
   file?: string;
   file_comment?: string;
+}
+
+export interface MessageEvent {
```

Comment:
As I mentioned above, you can remove this type definition if you have if/else in your message event listener.